lib/usbd/backend/usbd_stm32_otg_fs.c

```diff
@@ -75,10 +79,10 @@ static usbd_device *init(const usbd_backend_config *config)
 	_usbd_dev.config = config;
 
 	if (config->feature & USBD_VBUS_SENSE) {
-		if (OTG_FS_CID >= 0x00002000) { /* 2.0 */
+		if (OTG_FS_CID >= 0x00002000) { /* 2.0 HS core*/
```

Comment:
dow understand you. CID indicates an core version. according to RM0090 FS CID=0x0000 1100 and HS CID=0x0200 0600
what you ask about?